### ISSUE TITLE
Properties with all upper case character doesn't map

### ISSUE THREAD

**Issue by neverlieme** [MapsterMapper/Mapster#623]
Mapster doesn't map properties of two type with the same name and same type if all character of the property is uppercase. Is this some issue or it's a contract?

**Comment by SoranRad** [comment]
You can use GetMember method like this one 
```C#
TypeAdapterConfig
.GlobalSettings
     .GetMemberName((model, side) =>
     {
         if (side == MemberSide.Destination)
         {
             return model.Name.ToUpper();
            //or  return model.Name.ToLower();
         }
     }) ;
```

**Comment by andrerav** [comment]
I believe this is a duplicate of #370 which was resolved by #590. Please try the latest version on NuGet and let me know if it doesn't work for you.
